.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -3,35 +3,72 @@ name: Beta Feature Requirements
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the issue.)
+
+# Feature: 
+
+[//]: # (All information in this section is mandatory for new issue submission.)
+
+**Feature Name:**  
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary Lead(s):**
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
+
+**Short description:**
+
+[//]: # (A short description of the feature. One or two sentences maximum.)
 
-**Feature Name:** 
+**RFC:**
 
-**Design Doc:**
+[//]: # (This outlines the ideas for a feature and is used to collect early feedback. See the RFC template here: https://docs.google.com/document/d/1ewJoCcw5-04crH-M0xw4zFxz1cfwVCPnNyW4K3m4Yyc/template/preview)
 
-**Alpha Checklist:**
+**Design doc:**
+
+[//]: # (This is used for discussion of design and testing for a feature. See the design doc template here:https://docs.google.com/document/d/16FLQK8uhhic1ovKnnOG3OXJjFKs2aHnSmbximidpKwM/template/preview)
 
 **Relevant Documentation:**
 
+[//]: # (Links to documentation related to the feature. This may be tasks,
+# Beta Feature Requirements
+
+This page lists the requirements for promoting a feature to beta. Promotion to beta must meet all requirements for promotion to alpha. Please check off and document the steps as they are completed.
+
 --- 
 
 ### Requirements: 
 
 **Design**
 
-- [ ] Design doc describing the intention of the feature, how it will be
-	implemented, and any thoughts on how to test the feature has been approved by
-	relevant work group leads
+- [ ] Design doc has been approved by relevant work group leads
+
+[//]: # (A design doc is mandatory for graduation to beta. The design doc must be
+approved by relevant work group leads.)
+
 - [ ] Feature coverage and test plans written and approved.
+[//]: # (Feature coverage is required for graduation to beta. More details can be found here: https://github.com/istio/istio/blob/8af80bdf91684d8e517e9e5d57f2c3d8480b11a2/pkg/test/framework/features/README.md)
 
 **Docs** 
 
 - [ ] Documentation on istio.io includes performance expectations; may have caveats. 
 - [ ] Documentation on istio.io includes samples/tutorials. 
 - [ ] Documentation on istio.io includes appropriate glossary entries. 
 - [ ] All new documentation containing user actions includes istio.io tests.
+
+[//]: # (Documentation tests are required in order to be able to confirm that
+the documentation is correct across Istio releases. Details on writing Istio
+documentation tests can be found here: https://github.com/istio/istio.io/tree/master/tests)
+
 - [ ] Release notes have been added. 
 - [ ] Upgrade notes have been added. 
 
```

Comment:
below: how to determine if fixed issues have regression tests? manual scrub of all past issues?